src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -792,6 +794,13 @@ uct_ib_mlx5_devx_query_lag(uct_ib_mlx5_md_t *md, uint8_t *state)
     }
 
     *state = UCT_IB_MLX5DV_GET(lag_context, lag, lag_state);
+    lag_port_select_mode
+           = UCT_IB_MLX5DV_GET(lag_context, lag, port_select_mode);
+    if (*state && (lag_port_select_mode <= UCT_IB_LAG_MULTI_PORT_ESW)) {
+        *port_select_mode = (uct_ib_port_select_mode_t)lag_port_select_mode;
+    } else {
+        *port_select_mode = UCT_IB_LAG_INVALID_MODE;
+    }
```

Comment:
better do it outside of uct_ib_mlx5_devx_query_lag function - after it's called from uct_ib_mlx5_devx_md_open

Reply:
Done